src/getdim.f

```diff
@@ -1,33 +1,24 @@
+!>    @file
+!>    @brief This subroutine returns the info of GRIB2 Grid Definition
+!>    Section 3 format..
```

Comment:
@Hang-Lei-NOAA I am getting this doxygen warning:  _warning: End of list marker found without any preceding list items_

If I remove the extra period on line 3, the warning is eliminated.

Reply:
Yes, it is updated. Thanks!